src/main/java/view/OutputView.java

```diff
@@ -0,0 +1,54 @@
+package view;
+
+import domain.ladder.Ladder;
+import domain.LadderResult;
+import domain.ladder.Line;
+
+import java.util.List;
+
+public class OutputView {
+
+    public static void printLadder(Ladder ladder, List<String> playerNames, List<String> resultNames) {
+        System.out.println("실행 결과");
+
+        System.out.println(String.join(" ", playerNames));
+
+        for (Line line : ladder.getLines()) {
+            printLine(line);
+        }
+
+        System.out.println(String.join(" ", resultNames));
+    }
+
+    private static void printLine(Line line) {
+        StringBuilder result = new StringBuilder("|");
+
+        for (boolean point : line.getPoints()) {
+            if (point) {
+                result.append("----|");
+            } else {
+                result.append("    |");
+            }
```

Comment:
```suggestion
            if (point) {
                result.append("----|");
                continue;
            }
            result.append("    |");
```
이런 식으로도 else를 제거해볼 수 있겠네요!